example/FormatResp.py

```diff
@@ -57,7 +57,10 @@ def result_to_df(result: ResultSet) -> pd.DataFrame:
 
 def customized_cast_with_dict(val: ValueWrapper):
     _type = val._value.getType()
-    return getattr(_type, cast_as(_type))()
+    method = cast_as.get(_type)
+    if method != None:
```

Comment:
if method： is ok

Reply:
Sorry, I cannot get it. Could you please explain again?

Reply:
sorry what I means we can use some `Pythonic style`
```python3
# pythonic
a = []
if not a:
    do_something()
# not pythonic
if a != []:
    do_something()
# for None
# pythonic 
if a is not None:
    do_something()
# or
if not a:
    do_something()
# not pythonic
if a != None:
.....
```


link: https://docs.python-guide.org/writing/style/
but it is not a `must follow thing` but maybe better~?
![image](https://user-images.githubusercontent.com/15976103/184136742-3365dede-47d6-4be1-b9ef-6927937e5d25.png)

Reply:
@yihong0618 Thx! I update it~